corehq/apps/users/views/__init__.py

```diff
@@ -861,8 +884,11 @@ def _get_web_users(request, domains):
 @always_allow_project_access
 @require_can_edit_web_users
 @require_POST
+@location_safe
 def remove_web_user(request, domain, couch_user_id):
     user = WebUser.get_by_user_id(couch_user_id, domain)
+    if not user_can_access_other_user(domain, request.couch_user, user):
```

Comment:
Should we also throw this under the `if user:` conditional? Doesn't seem like `user_can_access_other_user` handles if `user` is None